tactics/tacticals.mli

```diff
@@ -219,11 +219,18 @@ module New : sig
     (Id.t -> unit Proofview.tactic) -> (Id.t -> unit Proofview.tactic) ->
     Id.t -> unit Proofview.tactic
 
+  val nthHypId : int -> Proofview.Goal.t -> Id.t
+  val nthHyp : int -> Proofview.Goal.t -> constr
```

Comment:
I know this is already used here and there but I am not a huge fan of the API using a `Proofview.Goal.t`. They are footguns w.r.t. the evarmap stored in the goal, I'd prefer a tactic assuming to be called on a focussed goal or an explicit env argument.

Reply:
OK, removed.

For the record, there are a few others in `tacticals.mli`, but my time is elapsed on this topic.